Xaml/Strings/zh-CN/Resources.resw

```diff
@@ -217,15 +217,15 @@
     <value>退出</value>
   </data>
   <data name="TrayFlyoutPage_LineButton.Text" xml:space="preserve">
-    <value>显示任务栏边框</value>
+    <value>显示任务栏细线</value>
   </data>
   <data name="TrayFlyoutPage_MaximizedWindow.Text" xml:space="preserve">
     <value>窗口最大化时</value>
-    <comment>Also used to generate the title of color picker windows</comment>
```

Comment:
This all looks great! One small detail: can you keep the comments in English? They don't show up to the user and only to the translator/developers.

Reply:
Done, thank you for ur reply